ds_judgements_public_ui/templates/pages/transactional_licence.html

```diff
@@ -7,7 +7,7 @@ <h3>How to apply</h3>
   <ol>
     <li>Read Part 1 &ndash; Instructions.</li>
     <li>
-      Answer all the relevant questions in Part 2 &ndash;<a href="https://caselaw.nationalarchives.gov.uk/static/documents/application_form_for_licence_to_re-use.docx"> Application Form.</a>
+      Answer all the relevant questions in Part 2 &ndash;<a href="https://caselaw.nationalarchives.gov.uk/static/documents/Find_Case_Law_Computational_Analysis_Licence_Application_Form.docx"> Application Form.</a>
```

Comment:
One very minor thing -- the bottom link (`{% static '/documents/...' %}`) has the advantage of linking to the staging file on staging, rather than the prod one. Not too worth worrying about though.